packages/terra-form-checkbox/tests/wdio/form-checkbox-spec.js

```diff
@@ -36,53 +31,46 @@ describe('Checkbox', () => {
         browser.click('#site');
       });
 
-      Terra.it.matchesScreenshot();
-      Terra.it.isAccessible();
+      Terra.it.validatesElement();
     });
   });
 
   describe('Disabled', () => {
     beforeEach(() => browser.url('/#/raw/tests/terra-form-checkbox/form-checkbox/checkbox/disabled-checkbox'));
 
```

Comment:
Is it possible to remove these beforeEach hooks? These can cause unexpected behaviors and slow down test execution time

Reply:
Fixed in cbac7ee370e2e95511fa85f71a127ab4a258ebec